.github/ISSUE_TEMPLATE/05-community-toolkit-docs-request.yml

```diff
@@ -0,0 +1,71 @@
+name: "Community Toolkit article request"
+description: Request a new article on a topic or concept related to the [Aspire Community Toolkit](https://github.com/CommunityToolkit/Aspire) that you can't find in our current docs.
+title: "[Community Toolkit New article]: "
+labels: ["doc-idea", "community-toolkit"]
+assignees: ["Alirexaa", "aaronpowell"]
+body:
+  - type: markdown
+    attributes:
+      value: "## Request a new article"
+  - type: markdown
+    attributes:
+      value: "Describe the proposed new article for us."
+  - type: input
+    id: topic
+    attributes:
+      label: Proposed topic or title
+      description: Write a short title or description of the topic
+    validations:
+      required: true
+  - type: markdown
+    attributes:
+      value: >
+        "Propose a location for the article. Use the `/` character to
+        define 'sub-folders' in the table of contents. Start with a top-level
+        guide, and follow through to the parent folder for the new article."
+  - type: input
+    id: toc-item
+    attributes:
+      label: Location in table of contents.
```

Comment:
This could be consolidated with the above question, both directly relate to the article placement.

Reply:
I don't think we need this do we? There's only a single place that we can put the articles for Community Toolkit

Reply:
Yeah, I agree. Just remove this please @Alirexaa